projects/storefrontlib/src/cms-components/checkout/components/delivery-mode/delivery-mode.component.ts

```diff
@@ -66,44 +79,29 @@ export class DeliveryModeComponent implements OnInit, OnDestroy {
         )
       )
       .subscribe(([deliveryModes, code]: [DeliveryMode[], string]) => {
-        if (!code && deliveryModes && deliveryModes.length) {
+        if (
+          !(
+            code &&
+            deliveryModes.find((deliveryMode) => deliveryMode.code === code)
```

Comment:
Since find doesn't return a boolean I would be incline to add a `!!` or `Boolean(...)`.

Reply:
Fixed.  Added `!!`